tests/src/test/scala/org/http4s/multipart/MultipartParserSpec.scala

```diff
@@ -260,14 +286,58 @@ object MultipartParserSpec extends Specification {
 
       val input = ruinDelims(unprocessedInput)
 
-      val results: Process0[Headers \/ ByteVector] = unspool(input) pipe MultipartParser.parse(boundary)
+      val results: Stream[Task, Either[Headers,Byte]] = unspool(input).through(MultipartParser.parse(boundary))
+
+      val (headers, bv) = results.runLog.unsafeRun().foldLeft((Headers.empty, ByteVector.empty)) {
+        case ((hsAcc, bvAcc), Right(byte)) => (hsAcc, bvAcc ++ ByteVector.fromByte(byte))
+        case ((hsAcc, bvAcc), Left(hs)) => (hsAcc ++ hs, bvAcc)
+      }
+
+      // I have Changed This Test.
+      // It previously only returned bar.
+      // What spec needs to be met that drops the first section?
```

Comment:
Because the previous test reset the accumulator when it received a header:

```scala
  case ((hsAcc, bvAcc), -\/(hs)) => (hsAcc ++ hs, ByteVector.empty)
```